R/util.R

```diff
@@ -2417,6 +2417,47 @@ week <- function(date, unit="year") {
   }
 }
 
+
+#' API to calculate duration between the start_date and the end_date in the provided time unit.
+time_between <- function(start_date, end_date=lubridate::today(), unit = "years") {
+  lubridate::time_length(lubridate::interval(start_date, end_date), unit = unit)
+}
+
+#' API to calculate duration between the start_date and the end_date in years.
+years_between <- function(start_date, end_date=lubridate::today()) {
+  time_between(start_date, end_date)
+}
+
+#' API to calculate duration between the start_date and the end_date in months.
+months_between <- function(start_date, end_date=lubridate::today()) {
+  time_between(start_date, end_date, unit = "months")
+}
+
+#' API to calculate duration between the start_date and the end_date in weeks.
+weeks_between <- function(start_date, end_date=lubridate::today()) {
+  time_between(start_date, end_date, unit = "weeks")
+}
+
+#' API to calculate duration between the start_date and the end_date in days.
+days_between <- function(start_date, end_date=lubridate::today()) {
+  time_between(start_date, end_date, unit = "days")
+}
+
+#' API to calculate duration between the start_date and the end_date in hours
+hours_between <- function(start_date, end_date=lubridate::today()) {
```

Comment:
Use now().

Reply:
Changed these to use `now()` as suggested.